api/graphql/schema.graphql

```diff
@@ -142,6 +142,39 @@ type Department {
     name: LocalizedString
 }
 
+enum Status {
```

Comment:
I would give this a more specific name, like "CandidateSearchStatus", to avoid conflicting with other kinds of status.